src/MoBi.Presentation/Tasks/Interaction/InitialConditionsTask.cs

```diff
@@ -1,14 +1,17 @@
 ﻿using System.Collections.Generic;
 using System.Linq;
+using System.Runtime.Remoting.Contexts;
```

Comment:
Check all the usings. You can use ReSharper to automatically clean up unnecessary ones. I'm sure this one and NPOI are not used, but they sometimes get added automatically by VS or ReSharper.